### PR TITLE
2018-01-05- My GCI experience.md

### PR DESCRIPTION
**Pull request by Ashesh3**
My  GCI submission blog post :D

### CODE REVIEW

**Comment by Ashesh3** [comment]
Updated the format and added content.
@yashovardhanagrawal

File: _posts/2018-01-06- My GCI experience.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+
+
```
Comment (yashovardhanagrawal) [inline _posts/2018-01-06- My GCI experience.md]: delete these 3 spaces, they are of no use

**Comment by Ashesh3** [comment]
@yashovardhanagrawal deleted.. done

**Comment by yashovardhanagrawal** [comment]
why did you close the PR? it was alright, just squash your commits

**Comment by Ashesh3** [comment]
@yashovardhanagrawal i messed up the original fork.. had to delete the whole repro.. Sorry

**Pull request closed without merging**